varia/print.scss

```diff
@@ -34,7 +34,6 @@ Andreas Hecht in https://www.jotform.com/blog/css-perfect-print-stylesheet-98272
   /* Fonts */
 
   body {
-    font: 13pt Georgia, "Times New Roman", Times, serif; // Fallback for browsers without CSS variables support.
```

Comment:
The postcss-custom-properties plugin will take care of adding the fallback rules for us.